src/tikzplotly/_dataContainer.py

```diff
@@ -2,7 +2,12 @@
 Contain the code to handle data in TikZ plots.
 """
 from ._utils import replace_all_digits, sanitize_text
-from ._data import post_treat_data
+from ._data import treat_data, post_treat_data
+
+def hexid_to_alpha(num):
```

Comment:
Maybe this is also due to my terrible rebase merge, but in some plot, the data is exported like that, with a digit then a letter, causing error at compilation
```tex
\pgfplotstableread{
x y0
1952 68.75
...
}\data0

\begin{tikzpicture}
\begin{axis}
\addplot+ [mark=none, solid, color=636efa, forget plot] table[y=y0] {\dataZ};
\end{axis}
\end{tikzpicture}
```
(this one has been obtained by grinning `test_line_charts`, but also occurs with `test_polar`)